### PR TITLE
mathvariant attributes should affect computed font style and weight in stylo

### PR DESCRIPTION
**Pull request by bzbarsky**
---

- [X] `./mach build -d` does not report any errors
- [X] `./mach test-tidy` does not report any errors
- [X] These changes fix https://bugzilla.mozilla.org/show_bug.cgi?id=1367301

- [ ] There are tests for these changes OR
- [X] These changes do not require tests because this is stylo-only and Gecko tests it.

---
This change is [Reviewable](https://reviewable.io/reviews/servo/servo/17026)

### CODE REVIEW

**Comment by bzbarsky** [comment]
@emilio want to review?

**Review by Manishearth — APPROVED** [review]

File: components/style/style_adjuster.rs [hunk 1]
```diff
@@ -7,8 +7,14 @@
...
 use app_units::Au;
 use properties::{self, ComputedValues, StyleBuilder};
+#[cfg(feature = "gecko")]
+use properties::longhands::_moz_math_variant::computed_value::T as moz_math_variant;
 use properties::longhands::display::computed_value::T as display;
 use properties::longhands::float::computed_value::T as float;
+#[cfg(feature = "gecko")]
+use properties::longhands::font_style::computed_value::T as font_style;
+#[cfg(feature = "gecko")]
+use properties::longhands::font_weight::computed_value::T as font_weight;
```
Comment (emilio) [inline components/style/style_adjuster.rs]: nit: when the stuff we use is conditionally compiled, if it's only used inside a function, I tend to just move it inside the function (see `adjust_for_text_combine_upright` for example).

Not a big deal in any case, it's fine to file an E-easy so it gets cleaned up in a followup.

**Review by emilio — APPROVED** [review]
Looks good

File: components/style/style_adjuster.rs (hunk 1 above)
Comment (bzbarsky) [inline components/style/style_adjuster.rs]: Good point.  Did that.

**Pull request merged**